Cargo.toml

```diff
@@ -53,7 +53,7 @@ harness = false
 
 [dependencies]
 arc-swap = "1.2"
-chrono = { version = "0.4", optional = true }
+chrono = { version = "0.4", optional = true, features = ["clock", "std"], default-features = false }
```

Comment:
@msrd0 - it looks like the "clock" feature already includes the "std" feature (https://docs.rs/crate/chrono/0.4.22/features).

```suggestion
chrono = { version = "0.4", optional = true, features = ["clock"], default-features = false }
```

Reply:
@msrd0 Can you apply this fix? @estk I can confirm that this fixes the CVE concern.